src/simpla-element-behavior.html

```diff
@@ -10,185 +10,218 @@
 
   window.SimplaBehaviors = window.SimplaBehaviors || {};
 
-  SimplaBehaviors.Element = {
-    properties: {
+  SimplaBehaviors.Element = function(config) {
+    let {
+          type,
+          getCallback,
+          setCallback,
+          dataProperties,
+          updateOn
```

Comment:
FWIW this is purely for `simpla-text` since it's `value` isn't a prop but a getter - I think it would be cleaner to enforce elements using this to set their values to legit props and just use observers. Is that viable for simpla-text?